Dockerfile

```
@@ -0,0 +1,22 @@
+FROM rust:1.79 as builder
+
+WORKDIR /app
+
+COPY ./ ./
+
+RUN cargo build --release
+
+FROM debian:bookworm-slim
+
+RUN apt-get update && \
+  apt-get upgrade && \
+  apt-get install -y libsqlite3-0 libpq5 && \
```

```suggestion
  apt-get install -y libsqlite3-0 libpq5 ca-certificates && \
```

We also need the SSL root certificates in case Esplora is used as chain backend